build.js

```diff
@@ -369,3 +369,19 @@ function validateMapItems(items, parentKeys) {
 function compareKeys(a, b) {
   return a.localeCompare(b, 'en', { sensitivity: 'base' });
 }
+
+/**
+ * Returns a short title to be used as the button label for linking to the given URL.
+ *
+ * @param {url} url The URL to which the button will link.
+ * @returns {string} A short title to be used as a button label.
+ */
+function titleForLink(url) {
+  const titlesForPrefixes = [
+    ['ably.com/docs', 'docs'],
+    ['faqs.ably.com', 'faq'],
+  ];
+
+  const foundPair = titlesForPrefixes.find((element) => (new RegExp(`https://${element[0]}/`)).test(url));
```

Comment:
I might be playing devil's advocate here because I haven't looked into where `element` comes from, but do we need to escape regex special characters?

Reply:
You've got me doubting now, so I'll go back and check again. The `element` is itself an array, where the first element (`0`) is the string to be matched (see the array defined above).

Reply:
I think the current strings don't need that, so I would rather not add code that's not actively needed in the current state of the code. If this is required in future then we can add it then.